iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/CoordinatedReactiveControlXmlSerializer.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.extensions;
+
+import com.google.auto.service.AutoService;
+import com.powsybl.commons.extensions.ExtensionXmlSerializer;
+import com.powsybl.commons.xml.XmlReaderContext;
+import com.powsybl.commons.xml.XmlUtil;
+import com.powsybl.commons.xml.XmlWriterContext;
+import com.powsybl.iidm.network.Generator;
+
+import javax.xml.stream.XMLStreamException;
+import java.io.InputStream;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+@AutoService(ExtensionXmlSerializer.class)
+public class CoordinatedReactiveControlXmlSerializer implements ExtensionXmlSerializer<Generator, CoordinatedReactiveControl> {
+
+    @Override
+    public boolean hasSubElements() {
+        return false;
+    }
+
+    @Override
+    public InputStream getXsdAsStream() {
+        return getClass().getResourceAsStream("/xsd/coordinatedReactiveControl.xsd");
+    }
+
+    @Override
+    public String getNamespaceUri() {
+        return "http://www.itesla_project.eu/schema/iidm/ext/coordinated_reactive_control/1_0";
```

Comment:
A general remark for future extension, we should stop using http://www.itesla_project.eu/ namespace. We have to keep it for old namespace including one providing from internal repo for backward compatibility but new extensions should use http://www.powsybl.org/...

Reply:
I will change the namespace. Do you think it’s better to use http or https?